src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java

```diff
@@ -463,13 +481,35 @@ private void updateCitizens()
     }
 
     /**
+     * Re-sorts the WorkOrders list according to the priorities inside the list.
+     */
+    private void sortWorkOrders()
+    {
+        workOrders.sort((first, second) ->
```

Comment:
This can be done much better:

workOrder.sort(Comparer.compare(WorkOrder::getPriority, Comparator.naturalOrder());
Will sort the Workorders, lowest priority first.

workOrder.sort(Comparer.compare(WorkOrder::getPriority, Comparator.reverse());
Will sort the WorkOrders, highest priority first. (So this is what you want ot use)